cfgov/teachers_digital_platform/jinja2/teachers_digital_platform/activity_search_facets_and_results.html

```diff
@@ -1,6 +1,6 @@
 {% import 'v1/includes/atoms/tag.html' as tag %}
 {% import 'v1/includes/molecules/pagination.html' as pagination with context %}
-{% from 'v1/includes/organisms/expandable.html' import expandable with context %}
+{% from 'tdp_expandable.html' import expandable with context %}
```

Comment:
```suggestion
{% from 'teachers_digital_platform/tdp_expandable.html' import expandable with context %}
```

If you don't mind, could we use the "full" path to templates instead of a relative one like this? The only reason it works without the full path is because we have [this custom code](https://github.com/cfpb/consumerfinance.gov/blob/7b6190d4c43c8fcffb5a126e1285ecb1e731344d/cfgov/v1/jinja2_environment.py#L14) which I'd like to remove someday.

Reply:
Absolutely! Good catch.